unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MinimapMetadataController.cs

```diff
@@ -24,13 +14,11 @@ public void Awake()
 
     public void UpdateMinimapSceneInformation(string scenesInfoJson)
     {
-        var scenesInfo = Utils.ParseJsonArray<MinimapSceneInfo[]>(scenesInfoJson);
-        foreach (var scene in scenesInfo)
+        var scenesInfo = Utils.ParseJsonArray<MinimapMetadata.MinimapSceneInfo[]>(scenesInfoJson);
+
+        foreach (var sceneInfo in scenesInfo)
         {
-            foreach (var parcel in scene.parcels)
-            {
-                minimapMetadata.SetTile(parcel.x, parcel.y, new MinimapMetadata.Tile(scene.type, scene.name));
-            }
+            minimapMetadata.AddSceneInfo(sceneInfo);
```

Comment:
should we clear this list before adding items to it?

Reply:
Not really, as scenes should accumulate over successive calls.